modules/custom/sql/mob_zones/02_cop/02_wilderness/024_lufaise_meadows.sql

```diff
@@ -247,10 +247,10 @@ REPLACE INTO `mob_spawn_points` (`mobid`, `spawnset`, `mobname`, `polutils_name`
     (16875761, 0,  'Leshy',                    'Leshy',                    66,  -235.774,  -15.743,   305.243,   205       ),
     (16875762, 0,  'Colorful_Leshy',           'Colorful Leshy',           67,  -183.619,  -15.623,   246.343,   217       ),
     (16875763, 0,  'Defoliate_Leshy',          'Defoliate Leshy',          68,  -162.788,  -16.185,   249.421,   23        ),
-    (16875764, 0,  'Fomor_Samurai',            'Fomor Samurai',            38,  -178,      -16.122,   304.7,     64        ),
+    (16875764, 0,  'Fomor_Samurai',            'Fomor Samurai',            92,  -178,      -16.122,   304.7,     64        ),
     (16875765, 0,  'Fomor_Thief',              'Fomor Thief',              61,  -180.6,    -15.888,   301.899,   224       ),
-    (16875766, 0,  'Fomor_Monk',               'Fomor Monk',               34,  -178.199,  -16.096,   299.299,   192       ),
-    (16875767, 0,  'Fomor_Ninja',              'Fomor Ninja',              33,  -175.5,    -15.782,   302.1,     160       ),
+    (16875766, 0,  'Fomor_Monk',               'Fomor Monk',               91,  -178.199,  -16.096,   299.299,   192       ),
+    (16875767, 0,  'Fomor_Ninja',              'Fomor Ninja',              90,  -175.5,    -15.782,   302.1,     160       ),
```

Comment:
Making them in the 90s?

Reply:
haha I was confused at that too, but he's changing the mob_group ID, not their level.